app/controllers/alaveteli_pro/public_bodies_controller.rb

```diff
@@ -3,7 +3,8 @@ class AlaveteliPro::PublicBodiesController < AlaveteliPro::BaseController
   def search
     query = params[:query] || ""
     xapian_results = typeahead_search(query, :model => PublicBody,
-                                             :exclude_tags => [ 'defunct' ])
+                                             :exclude_tags => [ 'defunct',
+                                                                'not_apply' ])
```

Comment:
Done in fa8eaf2 which also fixes an oversight in the specs that meant we weren't really testing the search results properly.